include/vicon/udp.h

```diff
@@ -326,8 +361,14 @@ class UDPClient
     ObjectDataType getObjectData(const std::string &name)
     {
         waitUntilConnected();
+
+        // Convert to fixed-size char array
+        BOB_ASSERT(name.size() < 24);
+        char bytes[24];
+        strcpy(bytes, name.c_str());
```

Comment:
Does the cmake build system define ``_CRT_SECURE_NO_WARNINGS`` on windows? If not, using, ``strcpy`` and its insecure friends results in compile errors.

Reply:
Yep I've defined it in the cmake file